### DISCUSSION TITLE
Bluetooth mouse with a Pico w?

### DISCUSSION THREAD

**Discussion by FlydGitHub** [micropython/micropython#11606] [RP2040 / Pico]
Bluetooth mouse with a Pico w?

I know this probably an odd request, but has anyone used a Bluetooth mouse with a Pico w?

Any suggestions appraised.

 Obmiouly my preference is for MircoPython code

**Comment by jimmo** [comment]
BLE is not currently supported on the Pico W.

We were waiting on fixes from Infineon to be integrated into the Pico SDK and the host stack. This is happening right now, so we will support this in the next release.

But you might be talking about Bluetooth Classic (not BLE), which is not supported at all in MicroPython.
